<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotterWindow</class>
 <widget class="QMainWindow" name="PlotterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>421</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>450</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	color: black;
   border: 2px solid orange;
	border-radius: 10px;
	background-color: white;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #eba171);
}</string>
    </property>
    <property name="text">
     <string>Calculate</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>450</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>-10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>510</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-99999.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100001.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black;
   border: 2px solid blue;
	border-radius: 10px;
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>X(min)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>500</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black;
   border: 2px solid blue;
	border-radius: 10px;
	background-color: white;
}</string>
    </property>
    <property name="text">
     <string>X(max)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>440</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black;
   border: 2px solid blue;
	border-radius: 10px;
	background-color: white;
}
</string>
    </property>
    <property name="text">
     <string>X(range)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>450</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_4">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>510</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000.000000000000000</double>
    </property>
    <property name="value">
     <double>10.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>500</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
	color: black;
   border: 2px solid blue;
	border-radius: 10px;
	background-color: white;
}
</string>
    </property>
    <property name="text">
     <string>Y(range)</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header location="global">qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
